src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -155,15 +149,42 @@
         // returns map of sobject name => list of fields
         var allFieldsBySObject = this.groupFieldsBySObject(everyField);
 
-        Object.keys(allFieldsBySObject).forEach(function(sObjectName) {
+        var fieldsBySObjectInOrder = {};
+
+        const OPPORTUNITY_SOBJECT_NAME = "Opportunity";
```

Comment:
I believe we don't use the BLOCK_CAPS naming standard for const in javascript, but instead use camel case / snake case as we would with other variable names - so "const opportunitySObjectName", etc